tasks/main.yml

```diff
@@ -87,6 +87,7 @@
         type: "{{ ad_integration_dns_connection_type }}"
         ip:
           dns: "{{ ad_integration_dns_server }}"
+        state: up
```

Comment:
Is this really necessary?  This makes the role not idempotent: https://github.com/linux-system-roles/network/#state-up
```
Note that state: up always re-activates the profile and possibly changes the networking configuration,
even if the profile was already active before. As a consequence, state: up always changes the system.
```

Reply:
From @liangwen12year in Slack:
For both initscripts provider and NM provider, if connection profile is not modified, then the previous activated connection will not be activated again (changed: false). The network role documentation should be updated.

I can verify that state: up does not result in changed status when run repeatedly